src/background/feature-flags-controller.ts

```diff
@@ -24,30 +24,27 @@ export class FeatureFlagsController {
         return this.featureFlagStore.getState();
     }
 
-    public disableFeature(feature: string): void {
-        const payload: FeatureFlagPayload = {
-            feature: feature,
-            enabled: false,
-        };
-        const message: Message = {
-            messageType: Messages.FeatureFlags.SetFeatureFlag,
-            payload: payload,
-            tabId: null,
-        };
-        this.interpreter.interpret(message);
+    public disableFeature(feature: string): FeatureFlagStoreData {
+        return this.toggleFeatureFlag(feature, false);
     }
 
-    public enableFeature(feature: string): void {
+    public enableFeature(feature: string): FeatureFlagStoreData {
+        return this.toggleFeatureFlag(feature, true);
+    }
+
+    private toggleFeatureFlag(feature: string, enabled: boolean): FeatureFlagStoreData {
         const payload: FeatureFlagPayload = {
-            feature: feature,
-            enabled: true,
+            feature,
+            enabled,
```

Comment:
Should we not have `feature: feature, enabled: enabled`? This seems to work fine but I am wondering why we don't have to be more explicit with it.

Reply:
Both approach has pros and cons. This is more clear and concise while it is easier to rename if using ````feature: feature, enabled: enabled````.